geonode/settings.py

```diff
@@ -1234,6 +1234,12 @@
 )
 """
 
+# To enable the WorldMap based Client enable those
+GEONODE_CLIENT_HOOKSET = "geonode.client.hooksets.WorldMapHookSet"
```

Comment:
I guess this should not be enabled by default, right?

Reply:
definitely. Sorry I meant to comment it out before pushing it. Will do it right now